src/pushcollector/_impl/collector.py

```diff
@@ -10,12 +10,34 @@ class Collector(object):
     Instances of this class shouldn't be constructed directly; rather,
     the :meth:`~pushcollector.Collector.get` method should be used to
     obtain a collector of the desired backend.
+
+    Custom backends *may* choose to implement the context manager protocol
```

Comment:
While I think this doc does a pretty good job of explaining the situation for the implementer of a backend, I think it's a bit confusing for someone who just wants to use this library.

Saying that a backend "may" support the context manager protocol makes it sound like it's not safe for me to do:

```python
    with Collector.get():
        ...
```

...without somehow knowing up-front whether the backend implements the context manager protocol.

It seems like it should be clarified somewhere that the returned value from Collector.get() can always be used as a context manager, regardless of the backend in use.  Maybe adding a sentence to the `get` doc string for that?

Reply:
Good point, I will update that as well.